README.md

```diff
@@ -4,115 +4,86 @@
 @adobe/aio-cli-plugin-asset-compute
 =======================
 
-Asset Compute Plugin for Adobe I/O Command Line Interface
+Asset Compute Plugin for the [Adobe I/O CLI](https://github.com/adobe/aio-cli). This plugin supports developing and testing of [Asset Compute custom workers](https://docs.adobe.com/content/help/en/asset-compute/using/extend/understand-extensibility.html).
 
 <!-- toc -->
 
 <!-- tocstop -->
 
-## Installation and Usage
+## Documentation
 
-This can be installed & used in three ways:
+Further documentation:
 
-1. [aio plugin](#install-as-aio-plugin)
-2. [local devDependency](#install-as-local-devdependency)
-3. [global standalone cli](#install-as-global-standalone-cli)
+- [Developing with `devtool`](https://docs.adobe.com/content/help/en/asset-compute/using/extend/develop-custom-application.html)
+- [Test framework `test-worker`](https://docs.adobe.com/content/help/en/asset-compute/using/extend/test-custom-application.html)
```

Comment:
we should also list `run-worker` here^

Reply:
It isn't mentioned in the public docs anywhere yet...

Reply:
ok...I think we should mention it. it comes up as an option when you use the `--help` flag:
<img width="562" alt="Screen Shot 2020-09-09 at 12 25 30 PM" src="https://user-images.githubusercontent.com/20048485/92644029-8dba4600-f297-11ea-87b8-65e7a6e2c055.png">

Reply:
oh nevermind, I see that the documentation for each cmd is mentioned further in the readme. This link is more for supporting documentation, maybe it should be called: `Adding custom tests with test framework`

from the description above it leads me to think it is explaining the test-worker cmd

Reply:
other than that, I thought the documentation looked good!

Reply:
Changed in 70387cc